HW_7/tlph_clients.py

```diff
@@ -0,0 +1,39 @@
+def get_name(id):
+    return input('Введите имя пользователя: ')
+
+def get_scnd_name(id):
+    return input('Введите ффамилию пользователя: ')
+
+def get_workplace(id):
+    return input('Введите место работы: ')
+
+def get_brth_date(id):
+    return input('Введите день рождения: ')
+
+def get_tlph_number(id):
+    j = int(input('Сколько телефонов?: '))
+    tlph = []
+    if j > 1:
+        for i in range(0, j):
```

Comment:
```suggestion
        for i in range(j):
```
Не обязательно первым аргументом функции `range` указывать `0` он передается по умолчанию.